### PR TITLE
Layout Things - Made the music elements expandable and fixed some other style things

### PR DESCRIPTION
**Pull request by ejohnson44**

### CODE REVIEW

File: fanClub/res/layout/fragment_playbar.xml [hunk 1]
```diff
@@ -1,34 +1,39 @@
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="fill_parent"
-    android:layout_height="match_parent"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
     android:orientation="horizontal"
-    android:background="@color/cornflower_blue">
+    android:background="#ffcccccc">
```
Comment (dgreenhalgh) [inline fanClub/res/layout/fragment_playbar.xml]: What was the reason for changing this to a harcoded string again? :hamburger:

Comment (ejohnson44) [inline fanClub/res/layout/fragment_playbar.xml]: I actually changed that - I just realized that I didn't merge master with this branch before I sent the pull request. I will do that when we get to Destin.

Comment (dgreenhalgh) [inline fanClub/res/layout/fragment_playbar.xml]: :-1:

Comment (ejohnson44) [inline fanClub/res/layout/fragment_playbar.xml]: I'm sorry.... I'm just now learning how branches work... :confused:

File: fanClub/res/values/dimensions.xml [hunk 2]
```diff
@@ -2,5 +2,6 @@
 <resources>
 
     <dimen name="show_content">80sp</dimen>
+    <dimen name="song_height">50sp</dimen>
 
 </resources>
\ No newline at end of file
```
Comment (theJenix) [inline fanClub/res/values/dimensions.xml]: Can you add a newline at the end of this file?

File: fanClub/src/com/lastcrusade/fanclub/components/PlaylistFragment.java [hunk 3]
```diff
@@ -51,7 +52,31 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
...
+    
+    private void toggleViewSize(View v){
+        TextView title = (TextView)v.findViewById(R.id.title); 
+        TextView album = (TextView)v.findViewById(R.id.album);
+        TextView artist = (TextView)v.findViewById(R.id.artist);
+        
+        if(v.getHeight()>getResources().getDimension(R.dimen.song_height)){
+            title.setMaxLines(1);
+            album.setMaxLines(1);
+            artist.setMaxLines(1);
```
Comment (theJenix) [inline fanClub/src/com/lastcrusade/fanclub/components/PlaylistFragment.java]: Magic numbers...please pull into an appropriately named constant or variable.

File: fanClub/src/com/lastcrusade/fanclub/components/PlaylistFragment.java [hunk 4]
```diff
@@ -51,7 +52,31 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
...
+            v.findViewById(R.id.user_color).setMinimumHeight((int) getResources().getDimension(R.dimen.song_height));
+        }
+        else{
+            title.setMaxLines(10);
+            album.setMaxLines(10);
+            artist.setMaxLines(10);
```
Comment (theJenix) [inline fanClub/src/com/lastcrusade/fanclub/components/PlaylistFragment.java]: Magic numbers...same as above.

File: fanClub/src/com/lastcrusade/fanclub/components/PlaylistFragment.java [hunk 5]
```diff
@@ -51,7 +52,31 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
...
+            
+            int viewHeight = (int) getResources().getDimension(R.dimen.song_height)
+                    + (title.getLineCount()-1)*title.getLineHeight()
```
Comment (theJenix) [inline fanClub/src/com/lastcrusade/fanclub/components/PlaylistFragment.java]: I might suggest pulling this calculation into a method, named based on it's function.  It's unclear what this is adding to the viewHeight.  Same for the next 2 lines.

File: fanClub/res/values/strings.xml [hunk 6]
```diff
@@ -19,5 +19,6 @@
     <string name="connect">Connect</string>
     <string name="no_fans_found">No fans found</string>    
     <string name="menu">Menu</string>
+    <string name="divider">-</string>
```
Comment (theJenix) [inline fanClub/res/values/strings.xml]: Is this intended to be used as is, or to be loaded and then copied n times?  A comment might help clarify this if we ever need to change it.

File: fanClub/src/com/lastcrusade/fanclub/components/PlaylistFragment.java [hunk 7]
```diff
@@ -24,9 +25,9 @@
     
     public PlaylistFragment(){
         for(SongMetadata s : metadataList){
-            s.setTitle("Title");
-            s.setAlbum("Album");
-            s.setArtist("Artist");
+            s.setTitle("This is a really really really looooooooooooooooooooooooooooooooooooooooooooooong Title");
```
Comment (theJenix) [inline fanClub/src/com/lastcrusade/fanclub/components/PlaylistFragment.java]: Wow that's really long.

File: fanClub/res/layout/song_item.xml [hunk 8]
```diff
@@ -5,23 +5,24 @@
...
-        android:id="@+id/userColor"
+        android:id="@+id/user_color"
         android:layout_width="10sp"
         android:layout_height="match_parent"
+        android:minHeight="@dimen/song_height"
         android:layout_alignParentLeft="true"
         android:layout_marginRight="4sp"
-        android:background="#ff000000"
-        android:minHeight="50sp" >
+        android:background="#ff000000" >
```
Comment (reidbaker) [inline fanClub/res/layout/song_item.xml]: Hardcoded color

File: fanClub/res/layout/song_item.xml [hunk 9]
```diff
@@ -5,23 +5,24 @@
...
     <TextView
         android:id="@+id/title"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_toRightOf="@+id/userColor"
-        android:paddingTop="2sp"
+        android:layout_toRightOf="@+id/user_color"
+        android:paddingTop="6sp"
+        android:maxLines="1"
         android:textColorHighlight="#656565"
```
Comment (reidbaker) [inline fanClub/res/layout/song_item.xml]: Hardcoded color

File: fanClub/res/layout/song_item.xml [hunk 10]
```diff
@@ -29,10 +30,24 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_below="@+id/title"
-        android:layout_toRightOf="@+id/userColor"
+        android:layout_toRightOf="@+id/user_color"
         android:paddingRight="2sp"
+        android:maxLines="1"
         android:textColorHighlight="#656565"
```
Comment (reidbaker) [inline fanClub/res/layout/song_item.xml]: Hardcoded color

File: fanClub/res/layout/song_item.xml [hunk 11]
```diff
@@ -29,10 +30,24 @@
...
+    
+     <TextView
+        android:id="@+id/division"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignBaseline="@+id/artist"
+        android:layout_toRightOf="@+id/artist"
+        android:paddingRight="2sp"
+        android:maxLines="1"
+        android:textColorHighlight="#656565"
```
Comment (reidbaker) [inline fanClub/res/layout/song_item.xml]: Hardcoded color

File: fanClub/res/layout/song_item.xml [hunk 12]
```diff
@@ -40,9 +55,10 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_alignBaseline="@+id/artist"
-        android:layout_toRightOf="@+id/artist"
+        android:layout_toRightOf="@+id/division"
+        android:maxLines="1"
         android:textColorHighlight="#656565"
```
Comment (reidbaker) [inline fanClub/res/layout/song_item.xml]: Hardcoded Color

File: fanClub/res/layout/song_item.xml [hunk 13]
```diff
@@ -50,9 +66,9 @@
         android:src="@drawable/av_add_to_queue"
         android:layout_width = "wrap_content"
         android:layout_height="wrap_content"
-        android:gravity="center_vertical"
+        android:layout_centerVertical="true"
         android:layout_alignParentRight="true"
         android:visibility="invisible"
-        android:background="@null"/>
+        android:background="#00000000"/>
```
Comment (reidbaker) [inline fanClub/res/layout/song_item.xml]: Hardcoded color

**Pull request merged**